adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -151,6 +151,16 @@ public Guid CorrelationId
             set { this.Authenticator.CorrelationId = value; }
         }
 
+#if iOS
+        /// Xamarin iOS specific property enables the application to share the token cache with other applications sharing the same keychain sharing group.
+        /// If you provide this key, you MUST add the capability to your Application Entitlement.
```

Comment:
pls. use regular xml comments. Also pls. add the remarks section saying that this API is subject to change.

Reply:
done.

---
In reply to: [208106186](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1171#discussion_r208106186) [](ancestors = 208106186)